tests/integration/test_client.py

```diff
@@ -63,11 +63,18 @@ async def test_client_start_transcribe_stream_bad_request(self, client):
 
     @pytest.mark.asyncio
     async def test_start_transcribe_stream_bad_boolean_show_speaker_lab(self, client):
-        with pytest.raises(SerializationException):
+        with pytest.raises(ValidationException):
             stream = await client.start_stream_transcription(
                 language_code="en-US",
                 media_sample_rate_hz=16000,
                 media_encoding="pcm",
                 show_speaker_label="foo",
                 enable_channel_identification=True,
             )
+
+        with pytest.raises(ValidationException):
```

Comment:
This is a separate test isn't it? It isn't related to the speaker boolean, so it probably shouldn't be bundled in here.